packages/terra-navigation-side-menu/src/MenuItem.module.scss

```diff
@@ -29,7 +29,15 @@
     z-index: 0;
 
     &:hover,
-    &.is-hovered,
+    &.is-hovered {
+      background-color: var(--terra-navigation-side-menu-item-hover-background-color, #f5fafe);
```

Comment:
just curious, i thought this PR was only supposed to contain rename changes?

Reply:
So this is just separating the same values into two different code blocks. A `&:hover, &.is-hovered` and a `&.is-focused:not(.is-selected):hover, &.is-focused:not(.is-selected).is-hovered` block. They share the same variables and default values. I believe this was to prevent needing to rename variables with extra values in their name but cannot seem to replicate whatever error caused the need for this to be split.